logstash-core/build.gradle

```diff
@@ -124,6 +124,20 @@ tasks.register("javaTests", Test) {
     exclude '/org/logstash/plugins/factory/PluginFactoryExtTest.class'
     exclude '/org/logstash/execution/ObservedExecutionTest.class'
 
+    if (runTestsInFIPSMode) {
```

Comment:
I think these test config modifications can live inside of x-pack: 

https://github.com/elastic/logstash/pull/17033

Reply:
this seems great! I just cherry-picked that commit to see how it does in CI. Thanks so much!